executor/oci/user.go

```diff
@@ -35,17 +39,23 @@ func GetUser(ctx context.Context, root, username string) (uint32, uint32, error)
 
 	execUser, err := user.GetExecUser(username, nil, passwdFile, groupFile)
 	if err != nil {
-		return 0, 0, err
+		return 0, 0, nil, err
 	}
-
-	return uint32(execUser.Uid), uint32(execUser.Gid), nil
+	var sgids []uint32
+	for _, g := range execUser.Sgids {
+		sgids = append(sgids, uint32(g))
+	}
+	return uint32(execUser.Uid), uint32(execUser.Gid), sgids, nil
 }
 
-func ParseUser(str string) (uid uint32, gid uint32, err error) {
+func ParseUIDGID(str string) (uid uint32, gid uint32, err error) {
 	if str == "" {
```

Comment:
is this behavior same in docker?

Reply:
@tonistiigi Nope. In docker, there was no fast path parsing from UID/GID directly.